### PR TITLE
[processing] python3/pyqt5 fixes for get scripts and models dialog

### PR DESCRIPTION
**Pull request by nirvn**
@volaya , @alexbruy , @m-kuhn , this PR resurrects the get scripts and models dialog, which has been broken since our transition to python3 / pyqt5.

### CODE REVIEW

File: python/plugins/processing/gui/GetScriptsAndModels.py [hunk 1]
```diff
@@ -194,8 +194,8 @@ def treeLoaded(self, reply):
         if reply.error() != QNetworkReply.NoError:
             self.popupError(reply.error(), reply.request().url().toString())
         else:
-            resources = str(reply.readAll()).splitlines()
-            resources = [r.split(',') for r in resources]
+            resources = str(bytes(reply.readAll()).decode('utf8')).splitlines()
```
Comment (m-kuhn) [inline python/plugins/processing/gui/GetScriptsAndModels.py]: I'd remove the cast-to-`str` part, `decode()` already returns that anyway

File: python/plugins/processing/gui/GetScriptsAndModels.py [hunk 2]
```diff
@@ -287,7 +287,7 @@ def storeFile(self, reply, filename):
                 self.popupError(reply.error(), reply.request().url().toString())
                 content = None
         else:
-            content = reply.readAll()
+            content = bytes(reply.readAll()).decode('utf8')
```
Comment (m-kuhn) [inline python/plugins/processing/gui/GetScriptsAndModels.py]: Just like here

**Comment by nirvn** [comment]
@m-kuhn , commit updated to address your comments; thanks.

**Comment by m-kuhn** [comment]
Thanks!

**Pull request merged**